clusim/clusimelement.py

```diff
@@ -267,7 +267,7 @@ def ppr_partition(clustering, alpha=0.9, relabeled_elements=None):
 
 
 def make_cielg(clustering, r=1.0, rescale_path_type='max',
-                relabeled_elements=None):
+                relabeled_elements=None, approach='previous'):
```

Comment:
I think the function body contains only the new approach? I think `sparse="true"` may be a better name?

Reply:
Yes, I'm always a big fan of keeping it simple, and sparse!

Reply:
Or, we can create another function `make_cielg_dense` or something.